sile/classes/quran.lua

```diff
@@ -183,7 +183,6 @@ SILE.registerCommand("aya", function (o,c)
     end)
   end)
   SILE.typesetter:typeset(o.text)
-  SILE.call("par") --- ?
```

Comment:
There should be a space in place of the `\par`. I tried `SILE.typesetter:typeset(o.text.." ")` and seems to be fine.